dwave/optimization/src/nodes/inputs.cpp

```diff
@@ -21,6 +21,30 @@
 
 namespace dwave::optimization {
 
+void check_limits(double min, double max) {
```

Comment:
Not that it hurts, by why break this out? Looks like it's only used in one place.

Reply:
I originally was using that in two constructors but then removed one. I'll move it back!

Reply:
Done